frontend/src/components/shared/ClipboardCopy.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { CDBInput } from 'cdbreact';
+import Button from "react-bootstrap/Button";
+import Container from "react-bootstrap/Container";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+export default function ClipboardCopy({ copyText }) {
+    const [isCopied, setIsCopied] = useState(false);
+
+    // This is the function we wrote earlier
```

Comment:
who is "we"? 🤔

Reply:
> You named this PR a chore, with the title "[chore(frontend): fix lose of session on signin](https://github.com/lunatech-labs/lunatech-chef/pull/234#top)" but you also added a new feature.


Yea, I did named it that when I initially started working on it, but along the line I noticed more clean up that wouldn't cost much to include with this PR. Also I just wanted to have a single PR so it would be easy for you while reviewing. I will update the PR name and also list the ISSUES related to this PR on the pr description